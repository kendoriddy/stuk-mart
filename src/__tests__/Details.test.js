import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import CompanyDetails from '../components/Details';
import store from '../redux/configureStore';

const DetailsProvider = () => (
  <Provider store={store}>
    <CompanyDetails />
  </Provider>
);

describe('renders properly the company details page', () => {
  it('renders correctly', () => {
    const component = renderer.create(<DetailsProvider />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
