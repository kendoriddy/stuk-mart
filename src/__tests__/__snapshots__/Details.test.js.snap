// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly the company details page renders correctly 1`] = `
<h2>
  Loading...
</h2>
`;
